packages/nitro-protocol/package.json

```diff
@@ -77,6 +79,9 @@
     "typescript": "3.6.3",
     "write-json-file": "4.2.0"
   },
+  "lint-staged": {
+    "{src,test}/**/*.tsx?": "yarn lint:check"
```

Comment:
Should this be `tsx` or `ts`?

Reply:
`tsx?` matches both `ts` and `tsx`. Are you asking if we shouldn't lint `tsx` files?